src/app/pipes/nodedisplay.ts

```diff
@@ -41,6 +41,22 @@ export class NodeTypePluralPipe implements PipeTransform {
 }
 
 
+@Pipe({name: 'nodeInternalUrl'})
+export class NodeInternalUrlPipe implements PipeTransform {
+    transform(node: any): string {
+        const baseUrl = 'https://pipeline.engineering.redhat.com/';
```

Comment:
We shouldn't hard code this URL here since other environments (e.g. dev and stage) have different URLs. Could you just do an HREF that excludes this? For instance, just `/${resourceType}/${identifier}` should work.